src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -754,14 +760,48 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         if (ptr.PointerDeviceType() == Windows::Devices::Input::PointerDeviceType::Mouse)
         {
-            if (point.Properties().IsLeftButtonPressed())
+            if (_terminal->IsSelectionActive() && point.Properties().IsLeftButtonPressed())
             {
                 const auto cursorPosition = point.Position();
-                const auto terminalPosition = _GetTerminalPosition(cursorPosition);
+                _SetEndSelectionPointAtCurstor(cursorPosition);
 
-                // save location (for rendering) + render
-                _terminal->SetEndSelectionPosition(terminalPosition);
-                _renderer->TriggerSelection();
+                const double cursorBelowBottomDist = cursorPosition.Y - _root.Padding().Top - _swapChainPanel.ActualHeight();
+                const double cursorAboveTopDist = -cursorPosition.Y + _root.Padding().Top;
+                constexpr double distThreashold = 2.0;

```

Comment:
```suggestion
                constexpr double distThreshold = 2.0;
```
Also, please add a comment for where this magic number came from,